packages/bff/.env

```diff
@@ -0,0 +1,6 @@
+HOST=0.0.0.0
+PORT=3100
+LOG_LEVEL=info
+
+WELL_KNOWN_URLS="https://dev.interop.pagopa.it/.well-known/jwks.json"
+ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui,refactor.dev.interop.pagopa.it/ui"
```

Comment:
It will be use in future?

Reply:
it's a requirement of the `JWTConfig` so without this values the service doesn't start